HLTrigger/special/src/HLTRechitsToDigis.cc

```diff
@@ -140,14 +140,17 @@ HLTRechitsToDigis::produce(edm::Event& iEvent, edm::EventSetup const& setup)  {
     // loop over the collection of rechits and match to digis
     EcalRecHitCollection::const_iterator ituneEB;
     for (ituneEB = recHitsHandle->begin(); ituneEB != recHitsHandle->end(); ituneEB++) {
-      EcalRecHit hit = (*ituneEB);
-      outputEBDigiCollection->push_back( (*digisEB->find(hit.id())).id(), (*digisEB->find(hit.id())).begin() );
+      EcalRecHit const &    hit = (*ituneEB);
+      EcalDigiCollection::const_iterator digiLookUp = digisEB->find(hit.id());
+      // protect against a digi not existing
+      if( digiLookUp == digisEB->end()) continue;
```

Comment:
Reviewing this PR now.... this check makes no sense, there are no recHits without a digi associated. The recHit should not be skipped as it is now.